From 0d4850405b74ac2a4b87522ad1ab8526f7e73127 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 30 Sep 2016 15:43:51 -0700
Subject: [PATCH] joyent/node-triton#136 bash completion for 'triton profile
 create --copy <TAB>' Reviewed by: Angela Fong <angela.fong@joyent.com>
 Approved by: Angela Fong <angela.fong@joyent.com>

---
 CHANGES.md                  | 1 +
 lib/do_profile/do_create.js | 5 +++--
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 56543bf..a9435d6 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -7,6 +7,7 @@ Known issues:
 
 ## not yet released
 
+- [#136] bash completion for `triton profile create --copy <TAB>`
 
 ## 4.14.0
 
diff --git a/lib/do_profile/do_create.js b/lib/do_profile/do_create.js
index 6a9f979..f5dcd60 100644
--- a/lib/do_profile/do_create.js
+++ b/lib/do_profile/do_create.js
@@ -352,8 +352,9 @@ do_create.options = [
     {
         names: ['copy'],
         type: 'string',
-        helpArg: 'NAME',
-        help: 'A profile from which to copy values.'
+        helpArg: 'PROFILE',
+        help: 'A profile from which to copy values.',
+        completionType: 'tritonprofile'
     },
     {
         names: ['no-docker'],
-- 
2.21.0

